### DISCUSSION TITLE
BeamDyn-AeroDyn coupling

### DISCUSSION THREAD

**Discussion by LaurenceWETI** [OpenFAST/openfast#1585] [Q&A] [answered]
Der @jjonkman,

I modified the BemaDyn input file for the NREL 5MW by adding 3 additional blade stations to NRELOffshrBsline5MW_BeamDyn.dat and NRELOffshrBsline5MW_BeamDyn_Blade.dat. The additional stations should help with specifying the distances between the key point in source code of BeamDyn as they are given in the  NRELOffshrBsline5MW_BeamDyn.dat. However, this modification causes the following error when AeroDyn in Test 26 is activated: 

![grafik](https://github.com/OpenFAST/openfast/assets/47774400/d4750bca-ba50-412c-b11e-d00f50190cc1)

Do the NRELOffshrBsline5MW_AeroDyn_blade.dat have to be mofied to be compatible with the modified input files of BeamDyn? Do you have any idea how to fix this error with keeping the additional stations in BeamDyn?

Best regards,

PS: the modified BeamDyn input files are attaches below:
[NRELOffshrBsline5MW_BeamDyn_modified.txt](https://github.com/OpenFAST/openfast/files/11532994/NRELOffshrBsline5MW_BeamDyn_modified.txt)
[NRELOffshrBsline5MW_BeamDyn_Blade_modfied.txt](https://github.com/OpenFAST/openfast/files/11532995/NRELOffshrBsline5MW_BeamDyn_Blade_modfied.txt)

**Comment by jjonkman** [comment, marked as answer]
Dear @LaurenceWETI,

You shouldn't need to change the AeroDyn discretization to match the BeamDyn discretization.   This error implies that your BeamDyn nodes are too closely spaced, leading to an error in the spatial mesh-mapping between BeamDyn and AeroDyn.  I haven't looked closely at your input file changes, but I would suggest adding a bit more space between the nodes for the new nodes you've added to the blade.

Best regards,
